### PR TITLE
 Commits 101

### PR DESCRIPTION
**Pull request by ketanprabhu-pk**
Commits 101

### CODE REVIEW

**Comment by ketanprabhu-pk** [comment]
Commits 101

**Review by ketanprabhu-pk — COMMENTED** [review]
comment 101

File: _posts/0000-01-02-KetanPrabhu97.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+[GitHub](http://github.com)
```
Comment (KetanPrabhu97) [inline _posts/0000-01-02-KetanPrabhu97.md:5]: GitHub

Comment (ketanprabhu-pk) [inline _posts/0000-01-02-KetanPrabhu97.md:5]: GitHub

**Review by ketanprabhu-pk — COMMENTED** [review]
GitHub

**Pull request merged**